Classes/Controller/Ajax/TranslationController.php

```diff
@@ -46,33 +47,33 @@ public function __construct(
 
     public function update(ServerRequestInterface $request): JsonResponse
     {
+        $this->assureModuleAccess();
+        $postParams = $request->getParsedBody();
+        $this->validateRequest($postParams);
+        $translationFile = $this->translationFileFactory->findByPath($postParams['path']);
+        $l10nTranslationFile = $translationFile->getL10nTranslationFile($postParams['language']);
+        $translation = new Translation($postParams['path'], $postParams['key'], $postParams['target']);
+        $l10nTranslationFile->upsertTranslationTarget($translation);
+        $this->translationFileWriterService->writeTranslation($l10nTranslationFile);
+        if ($postParams['language'] === 'default') {
+            $this->translationFileService->updateSourceInFiles($postParams);
+        }
+        $this->flushCache();
+        $content = [
+            'flashMessage' => [
+                'title' => 'OK',
+                'message' => 'label updated',
+                'severity' => AbstractMessage::OK,
+            ],
+        ];
```

Comment:
should this not be inside the try/catch block? (try block is empty?)

Reply:
ah thanks, I pulled it out of the try block to see the exception :)